util/docker/clang/Dockerfile

```diff
@@ -13,9 +15,16 @@ RUN apt-get update \
       python3-pip \
       git \
       gcovr \
- && wget https://apt.llvm.org/llvm.sh \
+      cmake \
+ && { apt-cache search clang-$llvmver | grep clang-$llvmver; } || { \
+    wget https://apt.llvm.org/llvm.sh \
  && chmod +x llvm.sh \
- && ./llvm.sh $llvmver all \
+ && ./llvm.sh $llvmver || ./llvm.sh $llvmver; } \
+ && apt-get install -y clang-${llvmver} lldb-${llvmver} lld-${llvmver} clangd-${llvmver} clang-tidy-${llvmver}  \
+    clang-format-${llvmver} clang-tools-${llvmver} llvm-${llvmver}-dev lld-${llvmver} lldb-${llvmver}  \
+    llvm-${llvmver}-tools libomp-${llvmver}-dev libc++-${llvmver}-dev libc++abi-${llvmver}-dev  \
+    libclang-common-${llvmver}-dev libclang-${llvmver}-dev libclang-cpp${llvmver}-dev libunwind-${llvmver}-dev \
+ && apt-get install clang-format-15 \
```

Comment:
This needs to be `clang-format-$llvmver`, otherwise apt cannot find clang-15 when building e.g. the image clang:18.

Reply:
`clang-format-15` is in the [debian bookworm package sources](https://pkgs.org/search/?q=clang-format-15), so this should build fine - maybe your cached version of `debian:latest` is outdated? The reason behind this is that I want a container that contains both a recent iwyu (using clang-18) and our clang-format-15, so that the CI can always apply both tools directly after each other.

Reply:
You were right, my `debian:latest` was outdated. I updated all the docker images now. Thank you!